Kickstarter-iOS/Views/Controllers/PledgeViewController.swift

```diff
@@ -98,6 +145,13 @@ final class PledgeViewController: UIViewController {
     NSLayoutConstraint.activate([
       self.rootStackView.widthAnchor.constraint(equalTo: self.rootScrollView.widthAnchor)
     ])
+
+    self.sectionSeparatorViews.forEach { view in
+      _ = view.heightAnchor.constraint(equalToConstant: 1)
```

Comment:
Did the design expect `1` or more hair like separators (which could usually be achieved by using `0.5` on retina devices (`1/UIDevice.screen.scale`)?

Reply:
The design specified 1 point for these separators.